File: Java/src/com/leetcode/Easy/SignOfTheProductOfAnArray.java
package com.leetcode.Easy;

// 1822. Sign of the Product of an Array
public class SignOfTheProductOfAnArray {
    public int arraySign(int[] nums) {
        int counter = 0;
        for (int num : nums) {
            if (num == 0)
                return 0;

            if (num < 0)
                counter++;
        }
        return counter % 2 == 0 ? 1 : -1;
    }
}

File: JavaScript/Easy/SignOfTheProductOfAnArray.js
/**
 * 1822. Sign of the Product of an Array
 * @param {number[]} nums
 * @return {number}
 */
var arraySign = function (nums) {
  let counter = 0;
  for (let i = 0; i < nums.length; i++) {
    if (nums[i] == 0) return 0;

    if (nums[i] < 0) counter++;
  }
  return counter % 2 == 0 ? 1 : -1;
};
